Kickstarter-iOS/Views/Controllers/SheetOverlayViewController.swift

```diff
@@ -2,6 +2,10 @@ import Library
 import Prelude
 import UIKit
 
+private enum SheetOverlayViewControllerStyles {
+  static let topAnchorMargin: CGFloat = 65
```

Comment:
This is used to add a buffer at the top of the screen to enable tapping to dismiss even when the font is quite large.

Reply:
I would add a comment here to say that `/// Allows tap to dismiss`, just because a few years from now it may not be clear as to intention, also helps our documentation.